taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def validate_license_number(license_number):
+    if len(license_number) != 8:
+        raise forms.ValidationError(
+            "license number must be 8 characters long"
+        )
+    if not license_number[3:8].isdigit():
+        raise ValidationError("Last 5 characters must be digits!")
+    if not license_number[:3].isalpha() or not license_number[:3].isupper():
+        raise ValidationError("First 3 characters must be uppercase letters!")
+    return license_number
+
+
+class DriverCreationForm(UserCreationForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[validate_license_number]
+    )
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The `drivers` field in `CarForm` uses `get_user_model().objects.all()` as its queryset. This might not be appropriate if `Driver` is a specific model that extends the user model. Consider using `Driver.objects.all()` instead to ensure only drivers are selectable.